starter-code/js/index.js

```diff
@@ -1,17 +1,43 @@
 function deleteItem(e){
-
+// var item = document.getElementByClass('wrapper');
+e.target.parentElement.parentElement.remove();
 }
 
-function getPriceByProduct(itemNode){
+// var input = document.querySelectorAll[i]('input');
+// input.addEventListener("keyup",)
 
-}
+// function getPriceByProduct(itemNode){
+//   var productCost = document.querySelectorAll('.cost');
+//   var productCount = document.querySelectorAll('.quantity');
+//   var grandTotal = parseInt(productCost[0].innerText) * parseInt(productCount[0].value);
+//   var total = document.querySelectorAll('.total');
+//   total[0].innerText = grandTotal.toFixed(2);
+// }
 
 function updatePriceByProduct(productPrice, index){
 
 }
 
 function getTotalPrice() {
```

Comment:
Instead of doing a very big-heavy function, break it in little functions that only do one thing.